components/automate-gateway/api/compliance/reporting/reporting.proto

```diff
@@ -308,6 +308,17 @@ message ControlItemRequest {
 	repeated ListFilter filters = 3;
 }
 
+message WaiverData {
+    // String identifying the waived state could be one of: yes, yes_run, no, no_expired
+	string waived_str = 1;
+	// The expiration date for the waiver.. after this date, the control is no longer waived
+	string expiration_date = 2;
+	// The reason for the waiver
```

Comment:
```suggestion
	// The reason for the waiver.
```

Reply:
Do the values of this follow any pattern? Is this a free form string provided by the author of the profile?